docker-compose.yml

```diff
@@ -166,7 +166,7 @@ services:
         container_name: plex
         image: plexinc/pms-docker:${PMSTAG}
         restart: unless-stopped
-        network_mode: "host"
+        network_mode: "bridge"
```

Comment:
This is the only change I'm iffy on -- I couldn't get "host" networking to work at all, after trying for a while last night. Switching it over the "bridge" worked instantly. I see you already have the port mappings needed for "bridge" mode -- was this intended to be bridge?

Reply:
Funny you mention this:

Reply:
Funny you mention this - I had been on the fence abut switching to "bridge" since I had the port mappings there, OR keeping it "host" and pulling the port mappings.

PLEX does appear to have their ports pretty set in stone - so bridge with mappings should hold fine for the foreseeable future.

Out of curiosity - does "host" work on a mac for other containers, and just not PLEX, or are you seeing the same behavior using "host" on any container?

Reply:
Great question -- I was under the assumption that "host" would work just fine on a Mac, because the Docker [documentation page ](https://docs.docker.com/engine/userguide/networking/#default-networks) doesn't mention anything about it not working.

However, I found this topic on the Docker forums: https://forums.docker.com/t/should-docker-run-net-host-work/14215 which shows "host" **does not work** as expected when using Docker for Mac. The post is from 2016, but I was able to reproduce the original submitter's observations using an up to date install. Interesting!

Reply:
Nice find - having not even used Docker for Mac but reading your post, I kind of thought that this might be a `--net=host` issue in general. 
Thanks for testing that theory for me.
That's kind of disappointing .. but as was mentioned in the thread not really a show stopper.